### PR TITLE
Add alternative Picosat Project

### PR DESCRIPTION
**Pull request by LuAbelt**
Additional fork for Picosat that does not rely on field sensitivity as much

### CODE REVIEW

File: varats/varats/projects/c_projects/picosat.py [hunk 1]
```diff
@@ -191,19 +191,19 @@ def get_release_revisions(
                 if re.match(release_regex, tag)]
 
 
-class PicoSATVaRA(VProject, ReleaseProviderHook):
+class PicoSATLT(VProject, ReleaseProviderHook):
```
Comment (vulder) [inline varats/varats/projects/c_projects/picosat.py]: For what does the `LT` stand?

**Review by vulder — APPROVED** [review]
LGTM

File: varats/varats/projects/c_projects/picosat.py (hunk 1 above)
Comment (LuAbelt) [reply varats/varats/projects/c_projects/picosat.py]: My initial thought was `Load Time` - I wanted to pick a more speaking addition to the normal Picosat for distinction that just `VaRA`, but in hindsight the `LT` itself is also not really more helpful.

Comment (vulder) [reply varats/varats/projects/c_projects/picosat.py]: The idea is good, maybe just call it that `PicoSATLoadTime`

File: varats/varats/experiments/vara/feature_perf_precision.py [hunk 2]
```diff
@@ -99,9 +99,9 @@ def get_threshold(project: VProject) -> int:
     if project.DOMAIN is ProjectDomains.TEST:
         if project.name in [
             "SynthSAFieldSensitivity", "SynthIPRuntime", "SynthIPTemplate",
-            "SynthIPTemplate2", "SynthIPCombined"
+            "SynthIPTemplate2", "SynthIPCombined", "PicosatLT"
```
Comment (LuAbelt) [inline varats/varats/experiments/vara/feature_perf_precision.py]:
```suggestion
            "SynthIPTemplate2", "SynthIPCombined", "PicoSATLoadTime"
```

File: varats/varats/projects/c_projects/picosat.py (hunk 1 above)
Comment (LuAbelt) [reply varats/varats/projects/c_projects/picosat.py]: Done.

Branch should be ready for a merge into vara-dev now

**Review by vulder — APPROVED** [review]

**Pull request merged**